docs/ruleset_functions/get_HVAC_systems_primarily_serving_comp_rooms.md

```diff
@@ -3,33 +3,32 @@
 **Description:** Returns a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.  
 
 **Inputs:**  
-- **U,B, or P-RMR**: To develop a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.    
+- **RMD**: To develop a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.    
 
 **Returns:**  
-- **hvac_systems_primarily_serving_comp_rooms_list_x**: A list of hvac systems in which greater than 50% of the area served by the HVAC system is computer room space.    
+- **hvac_systems_primarily_serving_comp_rooms_list**: A list of hvac systems in which greater than 50% of the area served by the HVAC system is computer room space.    
  
 **Function Call:**  
 
 1. get_hvac_zone_list_w_area()  
 
 ## Logic:   
-- Create RMR object from RMR function input (RMR = function input): `X_RMR = RMR`
-- Get dictionary list of zones and the total floor area served by each HVAC system: `hvac_zone_list_w_area_dict_x = get_hvac_zone_list_w_area (X_RMR)`
-- For each hvac_x in the X_RMR: `for hvac_x in X_RMR...HeatingVentilatingAirConditioningSystem:`
-    - Get the total floor area served by the hvac system: `hvac_sys_total_floor_area_x = hvac_zone_list_w_area_dict_x[hvac_x.id]["TOTAL_AREA"]`
+- Get dictionary list of zones and the total floor area served by each HVAC system: `hvac_zone_list_w_area_dict = get_hvac_zone_list_w_area(RMD)`
+- For each hvac_x in the X_RMD: `for hvac in RMD...HeatingVentilatingAirConditioningSystem:`
+    - Get the total floor area served by the hvac system: `hvac_sys_total_floor_area = hvac_zone_list_w_area_dict_x[hvac_x.id]["total_area"]`
     - Reset hvac system includes computer room boolean variable: `hvac_system_serves_computer_room_space = FALSE` 
     - Reset hvac_sys_computer_room_floor_area variable: `hvac_sys_computer_room_floor_area = 0`
-    - Get list of zones served by hvac_x: `hvac_zone_list_x = hvac_zone_list_w_area_dict_x[hvac_x.id]["ZONE_LIST"]`
-    - For each zone_x in hvac_zone_list_x: `for zone_x in hvac_zone_list_x:`
-        - For each space in zone: `for space_x in zone_x.Spaces:`        
-            - Check if space is of type computer room, if yes then set the hvac system has computer room boolean variable to true and add the floor area to the total computer room floor area variable for the hvac system: `if space_x.lighting_space_type in ["COMPUTER_ROOM"]:`
+    - Get list of zones served by hvac_x: `hvac_zone_list = hvac_zone_list_w_area_dict_x[hvac.id]["zone_list"]`
+    - For each zone_x in hvac_zone_list_x: `for zone in hvac_zone_list:`
```

Comment:
`For each zone_x in hvac_zone_list_x` same comment here

Reply:
Addressed